src/components/NavBar/NavBar.tsx

```diff
@@ -33,33 +37,42 @@ const styles = {
     height: "100%",
     marginLeft: "auto",
   },
+  list: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "10px",
+  },
 };
 
 const NavBar = () => {
+  const location = useLocation();
   const [openModal, setOpenModal] = useState(false);
   const { selectedConversation } = useAppSelector((state) => state.messages);
   const user = useAppSelector((state) => state.user);
 
   const navItems = [
     {
-      icon: <HomeIcon sx={styles.icon} />,
+      icon: <HomeOutlinedIcon fontSize="large" sx={styles.icon} />,
+      altIcon: <HomeIcon fontSize="large" sx={styles.icon} />,
```

Comment:
Minor nitpick, can we please rename this to `selectedIcon`?

Reply:
Done